andela/denethor-ah-backend
# 163478799 User should be able to create articles

#### What does this PR do?
Implement user can create an article

#### Description of Task to be completed?
* implement user can create new article

#### How should this be manually tested?
* clone this branch `feature/163478799/user-should-be-able-to-create-articles`
* signup a new user by hitting POST `/api/users` url
* retrieve the user token of the signed up user
* add the token to the authorization header with a bearer string. <Bearer ><token>
* create an article by hitting the POST `/api/articles`
  NB: if the token provided is invalid, the request will not go through otherwise, it would

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163478799

#### Screenshots (if appropriate)
None

#### Questions:
None
